docs/styles/demo/box-docs.less

```diff
@@ -7,23 +8,45 @@
     margin: 0; // FIXME
   }
 
-  .hxBox-xs {
+  .hxXs {
     box-shadow: inset 0 0 0 0.5rem @demo-padding-color;
   }
 
-  .hxBox-sm {
+  .hxSm {
     box-shadow: inset 0 0 0 0.75rem @demo-padding-color;
   }
 
-  .hxBox-md {
-    box-shadow: inset 0 0 0 1rem @demo-padding-color;
+  .hxMd {
+    box-shadow: inset 0 0 0 1.25rem @demo-padding-color;
   }
 
-  .hxBox-lg {
-    box-shadow: inset 0 0 0 1.25rem @demo-padding-color;
+  .hxLg {
+    box-shadow: inset 0 0 0 2rem @demo-padding-color;
+  }
+
+  .hxXl {
+    box-shadow: inset 0 0 0 3rem @demo-padding-color;
+  }
+
+  .hx2x {
+    box-shadow: inset 0 0 0 4.5rem @demo-padding-color;
   }
 
-  .hxBox-xl {
-    box-shadow: inset 0 0 0 1.5rem @demo-padding-color;
+  .hxFlush {
+    box-shadow: inset 0 0 0 0 @demo-padding-color;
+  }
+}
+
+.example > div.scroll-box-demo {
```

Comment:
may not need to be this specific

Reply:
Still needed, because I have to override example styling.